vocabularies/ODM.json

```diff
@@ -52,14 +51,32 @@
         "root": {
             "$Kind": "Term",
             "$Type": "Core.Tag",
-            "$Nullable": true,
             "$DefaultValue": true,
             "$AppliesTo": [
                 "EntitySet"
             ],
             "@Common.Experimental": true,
             "@Core.Description": "The annotated entity set contains root entities",
             "@Core.LongDescription": "Root Entities have an independent lifetime and are reachable as top level resources in APIs. These are usually the key entities of a domain, sometimes also called 'business objects'. They are called aggregate roots in DDD terminology."
+        },
+        "oid": {
+            "$Kind": "Term",
+            "$Type": "Edm.PropertyPath",
+            "$AppliesTo": [
+                "EntityType"
+            ],
+            "@Common.Experimental": true,
+            "@Core.Description": "The named field is an OID (ODM identifier), and it uniquely identifies an ODM root entity in a customer landscape.",
+            "@Core.LongDescription": "The technical type can be UUID or String (any length), and the value must be stable and unique at least within type and landscape (better globally). \nAn entity may have other IDs, i.e. a local ID and other alternative IDs. This is the ID used for ODM."
+        },
+        "entityName": {
+            "$Kind": "Term",
+            "$AppliesTo": [
+                "EntityType"
+            ],
+            "@Common.Experimental": true,
+            "@Core.Description": "Name of an ODM entity as a general concept, not a concrete version thereof",
+            "@Core.LongDescription": "This implies that the annotation cannot provide any contract about properties being contained in a model.\nThe annotated entity type thus is _not_ an implementation of the ODM entity, it only represents this ODM entity."
```

Comment:
@ralfhandl I am not sure about this description. With recent ODM additions, there are multiple representations of a single entity...

Reply:
not sure where you found this specific description and long description... But I missed that the xml is the source of truth of the json and md files